src/main/java/org/folio/dcb/client/feign/CirculationClient.java

```diff
@@ -1,14 +1,14 @@
 package org.folio.dcb.client.feign;
 
+import org.folio.dcb.client.feign.config.DcbClientConfiguration;
 import org.folio.dcb.domain.dto.CheckInRequest;
 import org.folio.dcb.domain.dto.CheckOutRequest;
 import org.folio.dcb.domain.dto.CirculationRequest;
-import org.folio.spring.config.FeignClientConfiguration;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 
-@FeignClient(name = "circulation", configuration = FeignClientConfiguration.class)
```

Comment:
as far as I see FeignClientConfiguration is responsible for introducing a number of beans into the app context, but the brand-new DcbClientConfiguration brings only ErrorDecoder.

I believe it's better to extend FeignClientConfiguration.
@MagzhanArtykov suggest, please.